v2/interface.go

```diff
@@ -58,6 +61,30 @@ type Logger interface {
 	// and writes to log with LevelCritical.
 	Critical(v ...any)
 
+	// TraceS writes a structured log with the given message and key-value
+	// pair attributes with LevelTrace to the log.
+	TraceS(ctx context.Context, msg string, attrs ...any)
```

Comment:
Since this is a new package and there's no compatibility concern, I wonder if we could make the structured log the default, so no more `DebugS` but only `Debugf`?

Reply:
ok nvm we need a `ctx` there